packages/app/lib/src/features/home/domain/post_entity.dart

```diff
@@ -16,10 +19,24 @@ sealed class PostEntity with _$PostEntity {
   ///
   /// Create a new, immutable instance of [PostEntity].
   const factory PostEntity({
+    /// The title of the post.
+    required String headline,
+
     /// The textual content of the post.
-    required String body,
+    required String description,
+
+    /// The author of the post.
+    required UserId author,
```

Comment:
> @lishaduck do we want classify users by their user id or email for posts? As of now I have it by email in appwrite but the posts class does it by user id. Which way do you think is gonna be better? 

 _Originally posted by @MattsAttack in [00dd7c4](https://github.com/PHS-TSA/nexus/commit/00dd7c4ae5426e55f6302e4a5bfe185e264ecf92#commitcomment-147671448)_

> I think user ID is probably a more robust solution. 

 _Originally posted by @lishaduck in [00dd7c4](https://github.com/PHS-TSA/nexus/commit/00dd7c4ae5426e55f6302e4a5bfe185e264ecf92#commitcomment-147678094)_